dap_aria_mapping/utils/novelty_utils.py

```diff
@@ -0,0 +1,255 @@
+"""
+Utils for measuring 'novelty' of research publication and patent abstracts
+"""
+from dap_aria_mapping import logging
+import pandas as pd
+import itertools
+import numpy as np
+from typing import Tuple
+
+
+def preprocess_topics_dict(
+    topics_dict: dict,
```

Comment:
doesn't matter too much, but we've been using Dict from typing so you can specify the types of the keys/values

Reply:
I see, makes sense, will fix.